lib/ValidateDirectiveVisitor.ts

```diff
@@ -919,15 +925,17 @@ abstract class ValidateDirectiveVisitor<
   // to validate the argument.
   public visitArgumentDefinition(
     argument: GraphQLArgument,
-    { field }: { field: GraphQLField<unknown, unknown> },
+    { field }: { field: GraphQLField<unknown, TContext> },
+    directiveArgs: TArgs,
   ): void {
     const validate = this.getValidationForArgs(
       DirectiveLocation.ARGUMENT_DEFINITION,
+      directiveArgs,
```

Comment:
i believe we can keep the `this.args` approach, why can't we keep using that?

Reply:
I'll still take a look on this one.